src/libraries/tests.proj

```diff
@@ -605,9 +630,16 @@
                       BuildInParallel="false" />
     <ProjectReference Condition="'$(RunGrpcTestsOnly)' == 'true'"
                       Include="@(GrpcTestProject)" />
+    <ProjectReference Condition="'$(TestReadyToRun)' == 'true'"
+                      Include="@(TestReadyToRunProjects)" />
   </ItemGroup>
 
-  <ItemGroup Condition="'$(RunSmokeTestsOnly)' != 'true' and '$(RunGrpcTestsOnly)' != 'true' and '$(RunHighAOTResourceRequiringTestsOnly)' != 'true' and '$(RunLimitedSetOfTests)' != 'true'">
+  <ItemGroup Condition="'$(RunSmokeTestsOnly)' != 'true'
```

Comment:
Please do not make this more complicated than it already is.

The existing `ProjectExclusions` ItemGroup should be exactly what you need here.

Reply:
Ok will change that. Just worth mentioning I thought you all didn't want to change that further because I originally disabled the bad tests with `ProjectExclusions`, and was told that was not the way to do it...

Reply:
> I originally disabled the bad tests with `ProjectExclusions`, and was told that was not the way to do it...

Can you point me to the comment asking not to use `ProjectExclusion`? I want to understand it.

Reply:
My bad, it was about disabling them in `System.Runtime.Tests.csproj`, not about `ProjectExclusions`. So many loops and very similar changes since this work was started I got confused for a second there.